R/merge_expression_module.R

```diff
@@ -1,5 +1,6 @@
 #' Merge expression module
 #'
+#' @description
 #' `r lifecycle::badge("experimental")`
 #'
 #' This function is a convenient wrapper to combine `data_extract_multiple_srv()` and
```

Comment:
This was previously part of `@details` hence the structure

I'd remove the start of the phrase.

```suggestion
#' Convenient wrapper to combine `data_extract_multiple_srv()` and
```